marklogic-data-hub-central/ui/e2e/hc-qa-project/build.gradle

```diff
@@ -49,3 +51,32 @@ task loadThesaurus(type: com.marklogic.gradle.task.MlcpTask) {
 }
 
 hubDeployAsDeveloper.finalizedBy("loadThesaurus")
+
+class DocCount extends HubTask {
+    @TaskAction
+    def getDocCount() {
+        int result
+        String script
+        def client
+        def propName = "database"
+        def database = project.hasProperty(propName) ? project.property(propName).toString() : null
+        if (database == null) {
+            throw new GradleException("Please specify a database via -Pdatabase=(name of staging or final database)")
+        } else if(database == "staging") {
```

Comment:
Instead of requiring a property to be specified, I recommend making two concrete tasks - e.g. verifyStagingCounts and verifyFinalCounts. Those are easier for a user to invoke and they have self-documenting task names. 

"script" then becomes a property of your custom task class. CallResourceTask in https://github.com/marklogic-community/ml-gradle/wiki/Writing-your-own-task is an example of that.